### PR TITLE
Attempt to describe how to pass toggles into production code

### PR DESCRIPTION
**Pull request by mrmanc**

### CODE REVIEW

File: exercises/java/src/main/java/toggle/BowlingCalculator.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import com.google.common.annotations.VisibleForTesting;
+
+public class BowlingCalculator {
+    private final boolean spareScoring;
+
+    public BowlingCalculator() {
+        this(false);
+    }
+
+    @VisibleForTesting
```
Comment (alectunbridge) [inline exercises/java/src/main/java/toggle/BowlingCalculator.java]: I wonder whether this annotation might just distract and confuse them, it did me.

Comment (mrmanc) [reply exercises/java/src/main/java/toggle/BowlingCalculator.java]: I’m going to talk through it and explain that there’s a constructor to allow the flag to be set, and a default which turns it off.

Comment (alectunbridge) [reply exercises/java/src/main/java/toggle/BowlingCalculator.java]: Yeah no problem with that, just it's another thing that they might struggle to grasp and think is essential to the feature flaggyness.  Not precious about it, happy for it to stay if you're comfortable with it.

Comment (mrmanc) [reply exercises/java/src/main/java/toggle/BowlingCalculator.java]: Fair point. Removing it.

**Pull request merged**